CHANGELOG.asciidoc

```diff
@@ -3,6 +3,24 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.17.15]]
+=== Beats version 7.17.15
+https://github.com/elastic/beats/compare/v7.17.14\...v7.17.15[View commits]
+
+==== Bugfixes
+
+*Metricbeat*
+
+- Fix CassandraConnectionClosures metric configuration {pull}34742[34742]
+
+==== Added
+
+*Affecting all Beats*
+
```

Comment:
@pierrehilbert shouldn't it at least mention the [upgrade of elastic-agent-libs](https://github.com/elastic/beats/commit/2a02e74caf4ada5e0c2319acde776e0f6e3139a0)?
From a quick look into the diff between the old and new version of elastic-agent-libs I believe the following are worth mentioning:


- https://github.com/elastic/elastic-agent-libs/commit/ef76bc6e4d942018de95fb6de731ee0d6258a160 fixes 
https://github.com/elastic/elastic-agent-libs/pull/113
- https://github.com/elastic/elastic-agent-libs/commit/a9b7e9bb97b4607048bfbd5c5ccc076b4916e8d1 changes a config key
- https://github.com/elastic/elastic-agent-libs/commit/4dea101389eae1e588e9d87aab5abcbb51171cee Fix IP Validation for TLS clients
- https://github.com/elastic/elastic-agent-libs/commit/734f0e1a6df00f6cc58d2ea9e06617f78f7660b3 fixes https://github.com/elastic/elastic-agent-libs/pull/156